client/src/components/Memories.js

```diff
@@ -0,0 +1,186 @@
+import '../styles/memories.css'
+import { React, useState, useEffect, memo} from 'react'
+import { handleGet, handlePost, handlePut } from '../services/requests-service'
+import {Snackbar} from '@mui/material'
+import {v4 as uuidv4} from 'uuid'
+
+export default function Memories() {
+  const [memories, setMemories] = useState([]);
+  
+  const [openSnackbar, setOpenSnackbar] = useState(false);
+  const [snackbarMessage, setSnackbarMessage] = useState("")
+
+  const getMemories = async() => {
+    const endpoint = `memories`
+    await handleGet(endpoint, setMemories)
+  }
+  useEffect(() => {
+    document.title = "Asti Memories"
+    getMemories();
+  }, [])
+  
+  const likePost = async(memory) => {
+    const endpoint = `memories`;
+    const requestBody = {
+      memory_uuid: memory.uuid,
+      num_likes: memory.num_likes + 1,
+    }
+    const response = await handlePut(endpoint, requestBody)
+    if(response.status === 200 || response.status === 201) {
+      getMemories();
+    } else {
+      alert("post like failed.")
+    }
+  }
+  const submitComment = async (comment, setNewComment) => {
```

Comment:
where is setNewComment defined or passed in from?

Reply:
and where is comment object being defined?

Reply:
The comment object is passed in from the submit comment method in the ViewComments mini-component